bridge/opencensus/metric.go

```diff
@@ -61,3 +62,42 @@ func (e *exporter) ExportMetrics(ctx context.Context, ocmetrics []*ocmetricdata.
 			},
 		}})
 }
+
+type reader struct {
+	// This reader doesn't actually matter. It is just there to satisfy the Reader interface.
+	metric.Reader
+	res     *resource.Resource
+	manager *metricproducer.Manager
+}
+
+// NewMetricExporter returns an OpenTelemetry metric.Reader that can be used as
+// a source of metrics for OpenTelemetry (pull) exporters.
+func NewMetricReader(res *resource.Resource) metric.Reader {
+	return &reader{
+		Reader: metric.NewManualReader(),
+		res:    res,
```

Comment:
The `manager` should also be initialized here otherwise it would fail at the `Collect` method where you call `producers := r.manager.GetAll()`